src/components/tables/Table.tsx

```diff
@@ -57,17 +57,22 @@ export interface ITableOwnProps extends React.ClassAttributes<Table>, ITableBody
     id: string;
     blankSlateDefault: IBlankSlateProps;
     tableContainerClasses?: string[];
+    tableClasses?: string[];
+    tableBodyClasses?: string[];
     initialTableData?: ITableData;
     actionBar?: true | IActionBarProps;
     blankSlateNoResultsOnAction?: IBlankSlateProps;
     blankSlateOnError?: IBlankSlateProps;
     datePicker?: IDatePickerDropdownProps;
     filter?: true | IFilterBoxProps;
-    filterMethod?: (attributeValue: any, props: ITableOwnProps) => boolean;
+    filterMethod?: (attributeValue: any, props: ITableOwnProps, filterValue: string) => boolean;

```

Comment:
give the param to check with the filter value if we want to add a custom filterMethod